meta-swupd/recipes-core/swupd-server/swupd-server_2.53.bb

```diff
@@ -30,6 +33,6 @@ do_install_append () {
     install -m 0755 ${S}/test/signature/* ${D}${sysconfdir}/swupd-certs/
 }
 
-RDEPENDS_${PN} = "tar rsync"
+RDEPENDS_${PN} = "libarchive rsync"
```

Comment:
Shouldn't this be bsdtar? That's what is added to the RDEPENDS for swupd-client above.